Lib/fontTools/feaLib/lexer.py

```diff
@@ -45,13 +47,13 @@ def __init__(self, text, filename):
     def __iter__(self):
         return self
 
-    def next(self):  # Python 2
-        return self.__next__()
+    def next(self, comments=False):  # Python 2
```

Comment:
an `iterator.__next__()` method cannot not take any extra argument (besides `self`):
https://docs.python.org/3.6/library/stdtypes.html#iterator.__next__